src/components/composer/composer/composer.ts

```diff
@@ -620,7 +630,14 @@ export class Composer extends Component<CellComposerProps, SpreadsheetChildEnv>
           break;
         case "REFERENCE":
           const { xc, sheetName } = splitReference(token.value);
-          result.push({ value: token.value, color: this.rangeColor(xc, sheetName) || "#000" });
+          result.push({
+            value: token.value,
+            color: this.rangeColor(xc, sheetName) || "#000",
+            class:
+              tokenAtCursor === token && this.props.composerStore.editionMode === "selecting"
+                ? referenceSelectedClass
```

Comment:
we could use the bootstap class `text-decoration-underline` directly here and remove the constant and the additional css

Reply:
> The first and third comments from Adrien are resolved but nothing changed.
definitely a miss on my hand, I'm positive I removed them ><